config/roguelike_dungeons/settings/loot_armor.json

```diff
@@ -0,0 +1,126 @@
+		{"data": {"name": "harvestthenether:quartzbootsItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "harvestthenether:quartzchestItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "harvestthenether:quartzhelmItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "harvestthenether:quartzleggingsItem", "meta": 0}, "weight": 10},
+		{"data": {"name": "adventurebackpack:adventureHat", "meta": 0}, "weight": 5},
+		{"data": {"name": "adventurebackpack:adventurePants", "meta": 0}, "weight": 5},
+		{"data": {"name": "adventurebackpack:adventureSuit", "meta": 0}, "weight": 5},
+		{"data": {"name": "adventurebackpack:pistonBoots", "meta": 0}, "weight": 5},
+		{"data": {"name": "minecraft:gold_boots", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:gold_chestplate", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:gold_helmet", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:gold_leggings", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_boots", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_chestplate", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 8},
+		{"data": {"name": "battlegear2:shield.hide", "meta": 0}, "weight": 15}
+    ]
+    },
+    {"level": 2, "type": "ARMOUR", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubychestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubyhelmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubyleggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubyboots", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphireboots", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphirechestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphirehelmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphireleggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_boots", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_chestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_helmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_leggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:iron_boots", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_chestplate", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 8},
+		{"data": {"name": "battlegear2:shield.gold", "meta": 0}, "weight": 15}
```

Comment:
battlegear2 is going to be removed, give out a fully built Tinker's Defense shield here instead.